src/main/java/com/epam/ta/reportportal/ws/controller/LaunchController.java

```diff
@@ -285,13 +285,14 @@ public OperationCompletionRS deleteLaunch(@PathVariable String projectName, @Pat
 	//		return updateLaunchHandler.startLaunchAnalyzer(normalizeId(projectName), launchId);
 	//	}
 	//
-	//	@RequestMapping(value = "/status", method = GET)
-	//	@ResponseBody
-	//	@ResponseStatus(OK)
-	//	@ApiIgnore
-	//	public Map<String, String> getStatuses(@PathVariable String projectName, @RequestParam(value = "ids") Long[] ids, @AuthenticationPrincipal ReportPortalUser user) {
-	//		return getLaunchMessageHandler.getStatuses(normalizeId(projectName), ids);
-	//	}
+	@RequestMapping(value = "/status", method = GET)
+	@ResponseBody
+	@ResponseStatus(OK)
+	@ApiIgnore
+	public Map<String, String> getStatuses(@PathVariable String projectName, @RequestParam(value = "ids") Long[] ids,
+			@AuthenticationPrincipal ReportPortalUser user) {
+		return getLaunchMessageHandler.getStatuses(normalizeId(projectName), ids);
```

Comment:
probably we should replace all normalizeId(projectName) with ProjectUtils.extractProjectDetails() method